python/cudf/cudf/core/series.py

```diff
@@ -307,6 +307,82 @@ def append(self, other, ignore_index=False):
 
         return Series._concat([this, other], index=index)
 
+    def clip(self, lower=None, upper=None, inplace=False):
+        """Trim values at input threshold(s).
+
+        Assigns values outside boundary to boundary values.
+        Thresholds can be singular values, and
+        in the latter case the clipping is performed element-wise.
+
+        Parameters
+        ----------
+        lower : Scalar value, default None
+            Minimum threshold value. All values below this
+            threshold will be set to it. If it is None,
+            there will be no clipping based on lower.
+        upper : Scalar value, default None
+            Maximum threshold value. All values above this
+            threshold will be set to it. If it is None,
+            there will be no clipping based on upper.
```

Comment:
Pandas docs say it can handle `array_like` inputs here.

Reply:
In this situation, you will have an array of elements with size equal to size of the series and each  row will have it's own `lower` and `upper`, but we don't support it in libcudf. 

I can add an exception saying not implemented.

Reply:
Ah, got it, thanks!